lib/fastlane/actions/carthage.rb

```diff
@@ -26,16 +21,13 @@ def self.description
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(key: :update,
```

Comment:
`command` option enables to support other commands without large modifying.
If someone wants to `carthage checkout`, he just need to modify the part `unless %w(build bootstrap update).include? value`.

Then, I added `update` as an optional value. https://github.com/uny/fastlane/commit/264c0133b3cdb0e88d2d79477eef4fd39a375a95
How about this?